entrypoint.sh

```diff
@@ -3,8 +3,8 @@
 set -e
 
 echo ${KUBE_CONFIG_DATA} | base64 -d > kubeconfig
-export KUBECONFIG=kubeconfig
+export KUBECONFIG="${PWD}/kubeconfig"
```

Comment:
@andyfoston Just for extra clarification, how would the dir change? Do we assume a different Dockerfile? Otherwise 
```
echo ${KUBE_CONFIG_DATA} | base64 -d > kubeconfig
```
in the line above will always export `kubeconfig` file in the current directory.

Thanks in advance and appreciate your contribution!